apps/site/src/_pages/docs/2_working-with-types.mdx

```diff
@@ -3,12 +3,12 @@
 ## Introduction
 
 When using the Block Protocol's [graph module](/docs/spec/graph-module), data is shared between applications and blocks
-in the form of 'entities', and each entity is assigned to at least one type, known as an 'entity type'.
```

Comment:
I know we'd like to extend the system in the future to allow multiple types per entity, but this is actively unsupported in the spec at the moment due to `entityTypeId` being a single string in the metadata of an `entity` right now. Opting to remove it to avoid confusion